src/EditorFeatures/Core/FindUsages/AbstractFindUsagesService_FindImplementations.cs

```diff
@@ -67,6 +67,11 @@ internal abstract partial class AbstractFindUsagesService
         private static async Task FindImplementationsInCurrentProcessAsync(
             ISymbol symbol, Project project, IFindUsagesContext context, CancellationToken cancellationToken)
         {
+            await context.SetSearchTitleAsync(
+                string.Format(EditorFeaturesResources._0_implementations,
+                FindUsagesHelpers.GetDisplayName(symbol)),
+                cancellationToken).ConfigureAwait(false);
```

Comment:
moved this to be first so that the title of the FAR window properly says `"IFoo" implementations` even if we end up finding nothing.